tests/integration/test_bulk_import_request.py

```diff
@@ -1,154 +1,141 @@
 import uuid
+from attr import validate
```

Comment:
It seems that `validate` isn't used in this file? Is this from the `attrs` library? It's not stated in the dependencies... Kinda confused what's going on here.